src/main/java/edu/utdallas/davisbase/parser/Parser.java

```diff
@@ -15,13 +34,204 @@ public Parser(ParserConfiguration configuration) {
+        return insert;
+      }
+      if (stmt instanceof Delete) {
+        Delete deleteStatement = (Delete) stmt;
+        DeleteCommandRepresentation delete = new DeleteCommandRepresentation(
+          deleteStatement.toString(),
+          deleteStatement.getTable().getName(),
+          parseWhereExpression(deleteStatement.getWhere())
+        );
+        return delete;
+      }
+      else if (stmt instanceof Update) {
+        Update updateStatement = (Update) stmt;
+        List<String> cols = new ArrayList<>();
+        for(Column c: updateStatement.getColumns()){
+          cols.add(c.getColumnName());
+        }
+        String vals = updateStatement.getExpressions().toString().replaceAll("[\\[''\\]]", "");
+        String[] splitValues=vals.trim().split("\\s*,\\s*");
+        UpdateCommandRepresentation update = new UpdateCommandRepresentation(
+          updateStatement.toString(),
+          updateStatement.getTables().get(0).getName(),
+          cols,
+          Arrays.asList(splitValues),
+          parseWhereExpression(updateStatement.getWhere())
+        );
+        return update;
+      }
+      else if(stmt instanceof Select){
+        Select selectStatement = (Select) stmt;
+        String selectBody = selectStatement.getSelectBody().toString();
+        String[] splitStr = selectBody.trim().split("SELECT |FROM ");
+        String[] splitValues=splitStr[1].trim().split("\\s*,\\s*");
+        Expression exp;
+        for(String col:splitValues){
+            try {
+              exp = CCJSqlParserUtil.parseExpression(col);
+              if(!(exp instanceof Column)){
+                throw new ParseException(exp + "is not a supported expression in Davisbase");
+              }
+            }catch(JSQLParserException e){
+              if(!col.equalsIgnoreCase("*")){
+                throw new ParseException(e);
+              }
+          }
+        }
+        SelectCommandRepresentation select = new SelectCommandRepresentation(
+          selectStatement.toString(),
+          splitStr[2],
+          Arrays.asList(splitValues),
+          splitValues[0].equalsIgnoreCase("*") ? true : false
+        );
+        return select;
+      }
+      else{
+        throw new ParseException("Sorry DavisBase does not support this command");
+      }
+    }
+    catch(JSQLParserException e){
+      throw(new ParseException(e.getCause()));
+    }
+    catch(ParseWhereException e){
+      throw(new ParseException(e.getCause()));
+    }
   }
 
+  /**
+   * @param where clause to parse
+   * @return WhereExpression representation of the expression
+   */
+  public WhereExpression parseWhereExpression(Expression where) throws ParseWhereException{
+    WhereExpression whereExpression;
+    if(where instanceof EqualsTo){
+      EqualsTo equals = (EqualsTo) where;
+      whereExpression= new WhereExpression(
+        equals.toString(),
+        equals.isNot(),
+        equals.getLeftExpression(),
+        equals.getStringExpression(),
```

Comment:
Let's either pass the already-structured `EqualsTo` object or--better yet--use some sort of our own structured enum (e.g. `...representation.ComparisonOperator.EQUALS_TO`) rather than a raw string for this.